### PR TITLE
Lunch menu

### PR DESCRIPTION
**Pull request by patkamon**
please comment to my work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## [Menu](Menu.md)
 
 
-[Breakfast](#breakfast) | [Lunch](#lunch) | [Dinner](#dinner) | [Night food](#night-food) | [Beverages](#beverages)
+[Breakfast](#breakfast) | [Lunch](/Menu.md#lunch) | [Dinner](#dinner) | [Night food](#night-food) | [Beverages](#beverages)
```
Comment (peach007) [inline README.md:6]: Should we be this style?

**Comment by pattanan-cyber** [comment]
Good did that very well

**Comment by vnsvakanda** [comment]
Good job

File: README.md (hunk 1 above)
Comment (patkamon) [reply README.md:6]: I already fix that, thx.

**Review by pattanan-cyber — APPROVED** [review]

**Comment by peach007** [comment]
good

**Review by HelloYeew — APPROVED** [review]
Good job!

**Comment by peach007** [comment]
good

**Pull request merged**